jerry-core/api/jerry.c

```diff
@@ -2759,6 +2759,81 @@ jerry_get_backtrace (uint32_t max_depth) /**< depth limit of the backtrace */
   return vm_get_backtrace (max_depth);
 } /* jerry_get_backtrace */
 
+/**
+ * Get the major API version defined in jerryscript-core.h
+ *
+ * @return the major API version
+ */
+uint32_t
+jerry_get_api_major_version (void)
+{
+  return JERRY_API_MAJOR_VERSION;
+} /* jerry_get_api_major_version */
+
+/**
+ * Get the minor API version defined in jerryscript-core.h
+ *
+ * @return the minor API version
+ */
+uint32_t
+jerry_get_api_minor_version (void)
+{
+  return JERRY_API_MINOR_VERSION;
+} /* jerry_get_api_minor_version */
+
+/**
+ * Get the component version described in parameter.
+ *
+ * Note:
+ *    * Debugger and snapshot versions are set to undefined, unless they are turned on.
```

Comment:
The comment states that the "debugger/snapshot version are set to undefined" if they are turned off. But in the code the return value will be 0. That looks pretty defined for me....